management-gateway/src/main/java/org/frankframework/management/bus/BusException.java

```diff
@@ -19,27 +19,43 @@
 import org.apache.logging.log4j.Logger;
 import org.frankframework.core.IbisException;
 
+import lombok.Getter;
+
 /**
  * Serialized and send as an ExceptionMessage over the Spring Bus
  */
 public class BusException extends RuntimeException {
-	private static final Logger LOG = LogManager.getLogger(BusException.class);
+	private static final transient Logger LOG = LogManager.getLogger(BusException.class);
```

Comment:
`static` is never serialized so does not need to be `transient` as well, to my knowledge.

(See also https://stackoverflow.com/questions/35083970/what-is-the-effect-of-static-final-transient-in-java and https://codezup.com/transient-keyword-in-java-static-vs-transient-vs-final/ )